xrpl/models/transactions/account_set.py

```diff
@@ -145,6 +145,60 @@ class AccountSet(Transaction):
     also set the AccountSetFlag.ASF_AUTHORIZED_MINTER flag.
     """
 
+    asf_account_tx_id: Optional[bool] = None
```

Comment:
This would be better implemented as a `TypedDict` of strings/enums to bools that is a part of `Flags`, instead of adding many additional instance variables to each class.

Reply:
Just to clarify. You mean something like this, right?
```python
class AccountSetFlagInterface(BoolFlags):  # 'BoolFlags' == 'TypedDict'
    asf_account_tx_id: bool
    asf_authorized_minter: bool
    asf_default_ripple: bool
    asf_deposit_auth: bool
    asf_disable_master: bool
    asf_disallow_xrp: bool
    asf_global_freeze: bool
    asf_no_freeze: bool
    asf_require_auth: bool
    asf_require_dest: bool
```